src/ui/preferences.ui

```diff
@@ -523,17 +523,17 @@ the application.</string>
         <normaloff>:/icons/wizard.svg</normaloff>:/icons/wizard.svg</iconset>
       </attribute>
       <attribute name="title">
-       <string>&amp;Scan Popup</string>
+       <string>&amp;Popup</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QCheckBox" name="startWithScanPopupOn">
          <property name="toolTip">
-          <string>Chooses whether the scan popup mode is on by default or not. If checked,
-the program would always start with the scan popup active.</string>
+          <string>Chooses whether the popup mode is on by default or not. If checked,
```

Comment:
Chooses whether the clipboard monitoring will be turned on by default on startup.